rover_msgs/include/rovus_lib/macros.h

```diff
@@ -1,3 +1,5 @@
+#include <ament_index_cpp/get_package_prefix.hpp>
+
```

Comment:
Must be after #ifndef to speed up compile time

Reply:
it has been moved